ofrak_components/Dockerstub

```diff
@@ -16,39 +16,6 @@ RUN apt-get -y update && \
       u-boot-tools \
       unar
 
-# Install CMake
```

Comment:
Can you document what this change is in the PR title/description?

Reply:
Also, is the Keystone install in `ofrak_core/Dockerstub` still needed?

Reply:
Documented the change in the description.

For now, the RBS Keystone build is still used in `ofrak_core`. I don't know if there will be test failures from switching to a `pip install keystone` setup, but it feels out of scope for this PR.